src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// write code here
+const addRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+addRow.onclick = () => {
+  const table = document.querySelector('table tbody');
+
+  const trCount = document.querySelectorAll('table tr').length;
+
+  if (trCount < 10) {
+    const template = document.querySelector('table tr');
+
+    const tdCount = template.children.length;
+
+    const tr = document.createElement('tr');
+
+    for (let i = 0; i < tdCount; i++) {
```

Comment:
Use functional programming constructs like `forEach` instead of `for` loops for iterating over arrays. Example: Array.from({length: tdCount}, () => tr.appendChild(document.createElement('td')));